src/components/Checkout/Checkout.js

```diff
@@ -8,10 +21,188 @@ const stripePromise = loadStripe(
 		''
 );
 
-export default function Checkout() {
+const Checkout = () => {
+	const [clientSecret, setClientSecret] = useState('');
+
+	useEffect(() => {
+		// Create PaymentIntent as soon as the page loads
+		axios
+			.post('http://localhost:4242/api/checkout/create-payment-intent', {
+				amount: 1000,
+			})
+			.then((res) => setClientSecret(res.data.clientSecret));
+	}, []);
+
+	const appearance = {
+		theme: 'stripe',
+	};
+	const options = {
+		clientSecret,
+		appearance,
+	};
+
 	return (
-		<Elements stripe={stripePromise}>
-			<PaymentForm />
-		</Elements>
+		<div>
+			{clientSecret && (
+				<Elements options={options} stripe={stripePromise}>
+					<Flex h='100vh' py={20}>
+						<SimpleGrid columns={3} rowGap={6} columnGap={3} w='full'>
+							<GridItem colSpan={2}>
+								<VStack
+									w='full'
+									h='full'
+									p={10}
+									spacing={10}
+									alignItems='flex-start'
+									style={{
```

Comment:
Let's not override styles this way. Use [these props](https://chakra-ui.com/docs/features/style-props) instead.